src/main/java/core/basesyntax/ArrayService.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class ArrayService {
+    private static final FigureSupplier supplier = new FigureSupplier();
+
+    public static void drawArrayElements(Figure[] figures) {
+        for (Figure figure : figures) {
+            figure.draw();
+        }
+    }
+
+    public static void fillingArray(Figure[] figures) {
```

Comment:
1. Do not use static for the moment except constants and Main class
2. naming
```suggestion
    private final FigureSupplier supplier = new FigureSupplier();

    public void drawArrayElements(Figure[] figures) {
        for (Figure figure : figures) {
            figure.draw();
        }
    }

    public void fillArray(Figure[] figures) {
```

Reply:
Ok